tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -97,7 +115,8 @@ public function testEnrollAgent() {
 
       // Test enrollment with bad token
       $agent = $this->agentFromInvitation($user, $guestEmail, $serial, 'bad token');
-      $this->boolean($agent->isNewItem())->isTrue(json_encode($_SESSION['MESSAGE_AFTER_REDIRECT'], JSON_PRETTY_PRINT));
+      $this->boolean($agent->isNewItem())->isTrue(json_encode($_SESSION['MESSAGE_AFTER_REDIRECT'],
+         JSON_PRETTY_PRINT));
```

Comment:
I disagree about hos the line is splitted. If you want split, then split by argument of isTrue() not in the expression of an argument.

Reply:
I think it was the auto formatting, probabbly because the line is to long to the 100 cols of the standard.